store/sql_channel_store.go

```diff
@@ -405,7 +405,7 @@ func (s SqlChannelStore) get(id string, master bool, allowFromCache bool) StoreC
 				if metrics != nil {
 					metrics.IncrementMemCacheHitCounter("Channel")
 				}
-				result.Data = cacheItem.(*model.Channel)
+				result.Data = (cacheItem.(*model.Channel)).DeepCopy()
```

Comment:
Mind if I ask why the copy is necessary? Does channelCache later overwrite the `Channel` pointer that it returned?